### DISCUSSION TITLE
I found a bug in the class PROJECT.change_log

### DISCUSSION THREAD

**Discussion by NomadSU** [princenyeche/jiraone#100] [General]
Hi
First of all, I have to admit that your product is good.
I found one error in PROJECT.change_log class
In the reporting.py file, on line 1061, you are trying to delete the "iter_saves.json" file, but this cannot be done because the file is already in use.
Well I found out why
On line 129 you open the file but then don't close it
i added os.close(fd) and now everything works for me

**Comment by princenyeche** [comment]
This has been fixed in #101 and #98 merge
